Makefile

```diff
@@ -12,6 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Running experiments locally.
+
+run-experiment: export COMPOSE_PROJECT_NAME := fuzzbench
```

Comment:
Does this require defining the target three times ?

Reply:
yep, this is how we can define variables that are local to the target (https://www.gnu.org/software/make/manual/html_node/Target_002dspecific.html)